documentation/runbooks/access-kubernetes.md

```diff
@@ -8,6 +8,8 @@ When you need to gain `kubectl` access to a platform-environments Kubernetes clu
 
 * An authenticated `az` cli. The logged in user must have permissions to list
   cluster credentials.
+* `docker` cli which is [authenticated against the GitHub Container Registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-to-the-container-registry).
```

Comment:
I'm not sure: did we still need this or did the need go away with the update to `.lagoon.yml` to no longer reference the regsitry?

Reply:
This is still needed to get [the `dplsh` image](https://github.com/danskernesdigitalebibliotek/dpl-platform/blob/main/tools/dplsh/dplsh.sh#L22).

Reply:
Ahh, got it. We should probably also add it to https://github.com/danskernesdigitalebibliotek/dpl-platform/blob/main/documentation/runbooks/using-dplsh.md then